### PR TITLE
Update grafana-metrics.md

### PR DESCRIPTION
**Pull request by nandlalyadav57**
admin info about Changing the Grafana Admin Password

### CODE REVIEW

File: software/grafana-metrics.md [hunk 1]
```diff
@@ -29,6 +29,36 @@ By default, users with [System Admin permissions](role-permission-reference.md#d
...
+    ```
+
+3. Run the following command to reset your Grafana password
+
+    ```bash
+    grafana-cli admin reset-admin-password '<new`-password>'
+    ```
+
+    Replace `<new-password>` with your desired new password.
+
```
Comment (jwitz) [inline software/grafana-metrics.md:58]:
```suggestion
```

**Review by jwitz — APPROVED** [review]
LGTM ! Just committed some grammar/ style updates, but the core content looks good

File: software/grafana-metrics.md [hunk 2]
```diff
@@ -29,6 +29,35 @@ By default, users with [System Admin permissions](role-permission-reference.md#d
...
+2. Run the following command to open the Grafana configuration folder:
+
+    ```bash
+    cd $GF_PATHS_HOME
+    ```
+
+3. Run the following command to reset your Grafana password
+
+    ```bash
+    grafana-cli admin reset-admin-password '<new`-password>'
```
Comment (stuart23) [inline software/grafana-metrics.md:54]: Is this stateful? When Grafana restarts, will it still apply?

File: software/grafana-metrics.md [hunk 3]
```diff
@@ -29,6 +29,35 @@ By default, users with [System Admin permissions](role-permission-reference.md#d
...
 
+
+### Change the Grafana admin password
+
+By default, the credentials for Grafana are `admin:admin`. To change the password:
+
+1. Run the following command to access the Grafana Pod in your Astronomer installation:
+
+    ```bash
+    kubectl exec -it <grafana-pod-name> bash -n astronomer
```
[the remark this answers is not in the document]
Comment (nandlalyadav57) [reply software/grafana-metrics.md:40]: Either should work

image

File: software/grafana-metrics.md (hunk 2 above)
Comment (nandlalyadav57) [reply software/grafana-metrics.md:54]: The method described for changing the admin password in Grafana using the CLI tool is stateful, meaning that the password change will persist even if Grafana restarts. Once you've reset the admin password using the provided steps, the new password will remain in effect until it is changed again using the same method.

File: software/grafana-metrics.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (nandlalyadav57) [reply software/grafana-metrics.md:40]: I realised it is better to use what you suggested as there is a warning of depreciation in the command

[the remark this answers is not in the document]
Comment (nandlalyadav57) [reply software/grafana-metrics.md:40]: https://github.com/astronomer/docs/pull/3627

[the remark this answers is not in the document]
Comment (nandlalyadav57) [reply software/grafana-metrics.md:40]: @jwitz https://github.com/astronomer/docs/pull/3627 can you merge this please

**Pull request merged**